include/os/linux/spl/sys/taskq.h

```diff
@@ -107,6 +107,25 @@ typedef struct taskq_sums {
 	wmsum_t tqs_thread_sleeps;		/* total thread sleeps */
 } taskq_sums_t;
 
+typedef struct taskq_time {
+	uint64_t tqtm_min;
+	uint64_t tqtm_max;
+	uint64_t tqtm_ewma_10;
+	uint64_t tqtm_ewma_100;
+	uint64_t tqtm_ewma_1000;
+	uint64_t tqtm_ewma_10000;
+} taskq_time_t;
```

Comment:
Did you mean to drop the timing stats from this PR?

Reply:
Yes, [per discussion upthread](https://github.com/openzfs/zfs/pull/16171#discussion_r1619240367).